Model/ResourceModel/ProductImage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Stockbase\Integration\Model\ResourceModel;
+
+/**
+ * Class StockItem
+ * @package Stockbase\Integration\Model\ResourceModel
+ */
+class ProductImage extends \Magento\Framework\Model\ResourceModel\Db\AbstractDb
+{
+
+    /**
+     * @var \Magento\Framework\EntityManager\EntityManager
+     */
+    protected $entityManager;
+
+    /**
+     * Constructor.
+     */
+    protected function _construct()
+    {
+        $this->_init('stockbase_product_images', 'id');
+    }
+
+    /**
+     * @param $imageName
+     * @param $productId
+     * @param $ean
+     * @return mixed
+     */
+    public function imageExists($imageName, $productId, $ean)
+    {
+        $connection = $this->getConnection();
+        $query = $connection->select()
+            ->from(['s' => $this->getMainTable()], ['s.ean'])
+            ->where('s.image = \''.$imageName.'\' AND s.product_id = '.$productId.' AND s.ean=\''.$ean.'\'');
```

Comment:
SQL Injection possible!
Please use prepared statement placeholder. See [the documentation](https://github.com/magento/magento2/blob/2.1/lib/internal/Magento/Framework/DB/Select.php#L77-L108) to the `where` method.